source/help/submit_tex.md

```diff
@@ -21,17 +21,17 @@
 <span id="newtex"></span>
 ### (La)TeX processing changes &mdash; coming Early 2025
 
-We will soon be rolling out changes to how arXiv process (La)TeX submissions. These changes should not be noticable to most of our users. We will be retiring the arXiv-developed "AutoTeX" system that we have used for decades in favor of a simpler, more straightfoward process of converting (La)TeX submissions to PDFs.
+We will soon be rolling out changes to how arXiv process (La)TeX submissions. These changes should not be noticable to most of our users. We will retire the long-used "AutoTeX" (Submission 1.0) system that we have used for decades in favor of a simpler, more straightfoward process of converting (La)TeX submissions to PDFs in Submission 1.5.
 
- 1. The AutoTeX system we are updating would try different versions of TeX to see which one successfully builds a PDF. From this point on, we will only use the version of TeX currently in use by arXiv.
+ 1. Submission 1.0 would try different versions of TeX to see which one successfully builds a PDF. From this point on, we will only use the version of TeX currently in use by arXiv.
     - Our plan is for arXiv's "current" version to closely follow the annual TeX Live releases. In the past, we often went several years between TeX updates.
- 1. The AutoTeX system we are updating would attempt to determine which files in a submission were part of the main document and which were not. We are no longer going to do this.
+ 1. Submission 1.0 would attempt to determine which files in a submission were part of the main document and which were not. We are no longer going to do this.
       - If there were multiple files ending in `.tex`, it would create PDFs for those extra files and append them to main paper's PDF. Now, if authors want a .tex file to be part of their main paper, they should use \include or \input commands to include the file. See [https://www.baeldung.com/cs/latex-include-vs-input](https://www.baeldung.com/cs/latex-include-vs-input).
```

Comment:
In the second sentence, should ".tex" get the same quotation treatment it gets elsewhere on this page?